docs/docs/extra-features.md

```diff
@@ -105,6 +105,25 @@ import React from 'react';
 export const Rating = ({ rating }) => <div>...</div>;
 ```
 
+[`babel-plugin-unassert`]: https://github.com/unassert-js/babel-plugin-unassert
+
+### Supported Assertion Function Names
+
+- `assert`
+- `invariant`
```

Comment:
Suggest flipping the order here if we feel that `invariant` is the preferred option, possibly adding a one liner as to why